# Service to get calibrated forces in hand
---
float64[] forces    # palm forces
int64 ret           # Set to 1 (success) or 0 (error)
string msg          # Error description
